README.md

```diff
@@ -907,3 +907,11 @@ harshithap2000-dai_deprivation"
 "orpheus_playing_hockey"
 
 ![](orpheus-playing-hockey.png)
+
+"when the X is just right dino"
+
+![](when_the_dino_is_just_right.jpg)
+
+"when the X is just right dino v2"
+
+![](when_the_dino_is_just_right_v2.jpg)
```

Comment:
Awesome drawing! The images are .png's though so the extension here needs to be changed from .jpg to .png.